releasenotes/notes/56877.yaml

```diff
@@ -7,7 +7,7 @@ issue:
 
 releaseNotes:
   - |
-    **Added** optional NetworkPolicy deployment for istiod
+    **Added** optional NetworkPolicy deployment for istiod, gateways and ztunnel.
```

Comment:
Was this added only for the main branch or was this previously released? If it was previously released (1.27 or earlier) then please make a new release note. Otherwise LGTM

Reply:
[b6622af](https://github.com/istio/istio/pull/57680/commits/b6622af9de90178695fd8abfd4a2f8f74ba18de8) yes, good catch, done